docs/modules/ROOT/pages/api/utilities.adoc

```diff
@@ -178,6 +186,86 @@ Returns the average of two unsigned integers. The result is rounded down.
 
 NOTE: `T` is a generic value matching different numeric implementations.
 
+[.contract]
+[[contract_clock]]
+=== `++contract_clock++`
+
+```cairo
+use openzeppelin_utils::contract_clock;
+```
+
+Module providing an interface for the {eip6372} standard along with default clock implementations based on either block number or block timestamp.
+
+[.contract-index]
+.Members
+--
+.Traits
+* xref:#IERC6372Clock[`++IERC6372Clock++`]
+--
+
+[.contract-index]
+.Impls
+--
+* xref:#contract_clock-ERC6372BlockNumberClock[`++ERC6372BlockNumberClock++`]
+* xref:#contract_clock-ERC6372TimestampClock[`++ERC6372TimestampClock++`]
+--
+
+[.contract]
+[#IERC6372Clock]
+=== `++IERC6372Clock++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v2.0.0-alpha.1/packages/utils/src/contract_clock.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_utils::contract_clock::IERC6372Clock;
+```
+
+Interface of the {eip6372} standard for exposing a contract's clock value and details.
```

Comment:
For consistency, we don't need a contract entry here showing the trait in the navbar to the right. We need a Trait block, explaining what the trait purpose is, similar to what we do with SerializedAppend in serde. The description should clarify that this is intenteded to be used with modules like Votes and Governor.